### PR TITLE
ISPN-6422 Use a synchronized list to store events

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-6422

### CODE REVIEW

**Comment by anistor** [comment]
Looking...

File: server/integration/event-logger/src/main/java/org/infinispan/server/eventlogger/ServerEventLogger.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ TimeService getTimeService() {
 
    @Override
    public List<EventLog> getEvents(Instant start, int count, Optional<EventLogCategory> category, Optional<EventLogLevel> level) {
-      List<EventLog> events = new ArrayList<>();
+      List<EventLog> events = Collections.synchronizedList(new ArrayList<>());
       AtomicReference<Throwable> throwable = new AtomicReference<>();
       try {
          cacheManager.executor().submitConsumer(m -> {
```
Comment (anistor) [inline server/integration/event-logger/src/main/java/org/infinispan/server/eventlogger/ServerEventLogger.java]: The throwable is wrapped in a CacheException and thrown. The outer try-catch(Exception) will wrap it again. I think we should have a separate catch for CacheException and Exception and avoid the double wrapping.

**Comment by anistor** [comment]
I made one comment, but otherwise looks good.

**Comment by tristantarrant** [comment]
Done @anistor

**Comment by anistor** [comment]
Integrated in master. Thanks @tristantarrant !

**Comment by anistor** [comment]
Backported to 8.2.x too.

**Pull request closed without merging**